pandas/tests/indexes/multi/test_compat.py

```diff
@@ -65,9 +65,7 @@ def test_inplace_mutation_resets_values():
     # ...and values is still same too
     tm.assert_almost_equal(mi1.values, vals)
 
-    # Inplace should drop _values from _cache
-    with tm.assert_produces_warning(FutureWarning):
-        mi1.set_levels(levels2, inplace=True)
+    mi1 = mi1.set_levels(levels2)
```

Comment:
Do we even need this part of the test any more? On initial glance seems duplicative of what's already in the test and if so would rather delete

Reply:
Good point, removed